test/credits/canonical_names_test.rb

```diff
@@ -2064,7 +2064,7 @@ class CanonicalNamesTest < ActiveSupport::TestCase
     end
 
     test "jeff\100ministrycentered.com" do
-      assert_contributor_names 'a5991d8', 'Jeff Berg'
+      assert_contributor_names 'a5991d849175c2ae9b803486f61b610fad9fd87e', 'Jeff Berg'
```

Comment:
**polish:** Based on the conversation at Rails World, I'm shortening the SHA to achieve the disambiguation and improved developer aesthetics.

```suggestion
      assert_contributor_names 'a5991d84', 'Jeff Berg'
```